tfdeterminism/patch.py

```diff
@@ -135,3 +142,110 @@ def _new_bias_add_1_14(value, bias, data_format=None, name=None):
           value, array_ops.reshape(bias, broadcast_shape), name=name)
     else: # data_format == 'NHWC' or data_format == None
       return math_ops.add(value, bias, name=name)
+
+
+def _patch_fused_softmax_cross_entropy():
+  # Sparse
+  tf.nn.softmax_cross_entropy_with_logits = _new_softmax_cross_entropy_with_logits_1_14  # access via public API
+  nn.softmax_cross_entropy_with_logits = _new_softmax_cross_entropy_with_logits_1_14  # called from tf.keras.layers.convolutional.Conv
+  nn_ops.softmax_cross_entropy_with_logits = _new_softmax_cross_entropy_with_logits_1_14  # called from tests
```

Comment:
Looking at [tensorflow/python/ops/nn_ops.py](https://github.com/tensorflow/tensorflow/blob/v2.2.0/tensorflow/python/ops/nn_ops.py#L3222) it seems like it might be necessary to remap the bindings for `softmax_cross_entropy_with_logits_v2` as well. After a brief review, I'm not sure what the difference between this and the other versions is, though, apart from the deprecated `dim` parameter.